tests/cbor_decode/test.sh

```diff
@@ -5,14 +5,14 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 
-for dir in 'test1_suit_old_formats/' 'test2_suit/' 'test3_simple/' 'test5_strange/' 'test7_suit9_simple' ;
-do
-        pushd "$dir"
+
+do_test() {
+        pushd "$1"
```

Comment:
Recommend to aid readability by making local variables in the function that are assigned to arguments. It's hard to know what `$1` means just from reading the function, whereas this can be a bit clearer:

```
    test_dir=$1
    board=$2
    
    pushd "$test_dir"
    ...
    cmake -GNinja -DBOARD=$board ..
```

Reply:
Thanks, done